Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm

```diff
@@ -614,7 +614,8 @@ static WallTime toSystemClockTime(NSDate *date)
         return;
 
     ensureOnMainThread([this, weakThis = WeakPtr { *this }] {
-        if (!weakThis)
+        auto protectedThis = RefPtr { weakThis.get() };
```

Comment:
Could I convince you that this pattern is more concise and looks nicer?
```
RefPtr protectedThis = weakThis.get();
```
Or are you really set on the other pattern? :)

Reply:
This looks great! Will adopt.